src/js/modules/modal-box.js

```diff
@@ -43,10 +41,10 @@ function closeModal(modal) {
 		gratitude.classList.remove("open");
 	}
 
-	modal.classList.remove("open");
-	document.removeEventListener("keydown", closeByEscape);
+	activeModal.close();
+	activeModal.removeEventListener("cancel", closeModal);
```

Comment:
it seems like you can remove it, if you use addEventListener in `modals.forEach((modal) => {`

Reply:
Done